init.js

```diff
@@ -49,6 +62,10 @@ module.exports = {
   // Derived fields are asynchronous functions that are given the previous user
   // input data of the form: `function (data, cb)`. They callback with:
   // `(err, value)`.
+  //
+  // **Note**: We have implied derived values of:
+  // - `{{npmignore}}` -> `.npmignore`
+  // - `{{gitignore}}` -> `.gitignore`
```

Comment:
I don't fully understand these lines, are these defaults?. Does this allow archetype authors to define `.npmignore` and `.gitignore` files?